docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -241,7 +241,7 @@ const buildComponentDocumentation = async (options: {
   reactApi.styles = await parseStyles(reactApi, project.program as any);
   reactApi.styles.name = reactApi.name.startsWith('Grid')
     ? 'MuiDataGrid' // TODO: Read from @slot annotation
-    : `Mui${reactApi.name.replace(/Pro$/, '')}`;
+    : `Mui${reactApi.name.replace(/((Pro)|(Premium))$/, '')}`;
```

Comment:
It's not necessary to wrap the alternatives in parenthesis.

```suggestion
    : `Mui${reactApi.name.replace(/(Pro|Premium)$/, '')}`;
```